cmd/virt-handler/virt-handler.go

```diff
@@ -388,6 +396,9 @@ func (app *virtHandlerApp) AddFlags() {
 	flag.StringVar(&app.VirtPrivateDir, "kubevirt-private-dir", util.VirtPrivateDir,
 		"private directory for virt-handler state")
 
+	flag.StringVar(&app.NetworkInfoDir, "network-info-dir", util.NetworkInfoDir,
```

Comment:
I don't think we need this to be a cli option right now. If someone wants to change the base dir, they can change the 'kubevirt-private-dir" variable.